src/libcommonserver/utility/utility.cpp

```diff
@@ -913,4 +913,9 @@ std::string Utility::userName() {
     return userName_private();
 }
 
+ExitInfo Utility::terminateThreadFunction(ExitInfo exitInfo) {
```

Comment:
As it is, this method does not change the value of `exitInfo`. Do you envision in the future more code that could modify this value? If not, it would be better to not pass and to not return any `exitInfo`.

Reply:
This way, we gain on concision (one less line), but we lose on clarity: the signature of the call suggests that the passed `ExitInfo` could be modified based on the failure or success of the termination process, but it seems unrelated.

Reply:
"const" added to the parameter.
Documentation added.

Reply:
StdLoggingThread and QtLoggingThread classes added.
FolderWatcher._thread and ISyncWorker._thread are still std::thread because their construction involves "this".